backend/danswer/db/chat.py

```diff
@@ -202,6 +250,40 @@ def get_chat_messages_by_session(
     return list(result)
 
 
+class ChatMessageSkeleton:
+    message_id: int
+    chat_session_id: int
+    time_sent: datetime.datetime
+
+    def __init__(
+        self, message_id: int, chat_session_id: int, time_sent: datetime.datetime
+    ) -> None:
+        self.message_id = message_id
+        self.chat_session_id = chat_session_id
+        self.time_sent = time_sent
+
+
+# Gets skeletons of all message
+# TODO: should change this to use a key index with dates
+# TODO: Need to paginate as well
+def get_empty_chat_messages_entries(
+    db_session: Session,
+) -> list[ChatMessageSkeleton]:
+    stmt = select(
+        ChatMessage.id, ChatMessage.chat_session_id, ChatMessage.time_sent
+    ).where(ChatMessage.message_type == MessageType.USER)
+
+    result = db_session.execute(stmt).all()
+
+    print(result)
```

Comment:
We prefer not to use print in any of the main flows, we use the standard logger that we have throughout the code base.

Reply:
oops good catch